### PR TITLE
Adding CITATION.cff file

### PR DESCRIPTION
**Pull request by gvwilson**
Please add your ORCID as a comment and I'll merge.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,47 @@
+cff-version: 1.2.0
+message: "Please cite this paper as shown."
+authors:
+- family-names: "Irving"
+  given-names: "Damien"
+  orcid: ""
```
Comment (DamienIrving) [inline CITATION.cff]:
```suggestion
  orcid: "https://orcid.org/0000-0003-1258-5002"
```

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- family-names: "Wilson"
+  given-names: "Greg"
+  orcid: "https://orcid.org/0000-0001-8659-8979"
+title: "Research Software Engineering with Python: Building Software that Makes Research Possible"
+preferred-citation:
+  type: book
+  authors:
+  - family-names: "Irving"
+    given-names: "Damien"
+    orcid: ""
```
Comment (DamienIrving) [inline CITATION.cff]:
```suggestion
    orcid: "https://orcid.org/0000-0003-1258-5002"
```

File: CITATION.cff [hunk 3]
```diff
@@ -0,0 +1,47 @@
+cff-version: 1.2.0
+message: "Please cite this paper as shown."
+authors:
+- family-names: "Irving"
+  given-names: "Damien"
+  orcid: "https://orcid.org/0000-0003-1258-5002"
+- family-names: "Hertweck"
+  given-names: "Kate"
+  orcid: ""
```
Comment (k8hertweck) [inline CITATION.cff]:
```suggestion
  orcid: "https://orcid.org/0000-0002-4026-4612"
```

File: CITATION.cff [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+title: "Research Software Engineering with Python: Building Software that Makes Research Possible"
+preferred-citation:
+  type: book
+  authors:
+  - family-names: "Irving"
+    given-names: "Damien"
+    orcid: "https://orcid.org/0000-0003-1258-5002"
+  - family-names: "Hertweck"
+    given-names: "Kate"
+    orcid: ""
```
Comment (k8hertweck) [inline CITATION.cff]:
```suggestion
    orcid: "https://orcid.org/0000-0002-4026-4612"
```

File: CITATION.cff [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  orcid: "https://orcid.org/0000-0002-4026-4612"
+- family-names: "Johnston"
+  given-names: "Luke"
+  orcid: ""
+- family-names: "Ostblom"
+  given-names: "Joel"
+  orcid: ""
+- family-names: "Wickham"
+  given-names: "Charlotte"
+  orcid: ""
```
Comment (cwickham) [inline CITATION.cff]:
```suggestion
  orcid: "https://orcid.org/0000-0002-6365-5499"
```

**Review by cwickham — COMMENTED** [review]
Adding orcid

**Pull request merged**